sdk/src/Core/Amazon.Runtime/Credentials/Internal/AwsV4aAuthScheme.cs

```diff
@@ -36,7 +37,9 @@ public IIdentityResolver GetIdentityResolver(IIdentityResolverConfiguration conf
         /// <inheritdoc/>
         public ISigner Signer()
         {
-            return new AWS4aSignerCRTWrapper();
+            if (_signer == null)
```

Comment:
Any reason why this one doesn't assign the value where the field is declared? Is it because it'll try to load the CRT dependency?

Reply:
Hm, that makes me think we should update the `BaseAuthResolverHandler` to also check if the CRT is available when there are multiple auth schemes and one of them is SigV4A (in that case, we should skip SigV4A and try to use one of the other auth schemes).

This can be done in a separate PR; Here's what the endpoints resolver does: https://github.com/aws/aws-sdk-net/blob/816c44e7c6a19639cc65065c0f8fd533420638b0/sdk/src/Core/Amazon.Runtime/Pipeline/Handlers/BaseEndpointResolver.cs#L166-L176